vividus-plugin-azure-storage-account/src/main/java/org/vividus/azure/storage/blob/BlobStorageSteps.java

```diff
@@ -59,7 +59,7 @@
                 @Override
                 public BlobServiceClient load(String endpoint)
                 {
-                    return new BlobServiceClientBuilder().endpoint(endpoint).credential(credential).buildClient();
+                    return new BlobServiceClientBuilder().credential(credential).endpoint(endpoint).buildClient();
```

Comment:
What is the reason for re-ordering here?

Reply:
1.
<img width="788" alt="Screen Shot 2021-05-19 at 9 58 40 AM" src="https://user-images.githubusercontent.com/5081226/118772295-43f51700-b88c-11eb-9a2a-8898f9d68ea4.png">
2.
<img width="509" alt="Screen Shot 2021-05-19 at 9 59 16 AM" src="https://user-images.githubusercontent.com/5081226/118772308-48b9cb00-b88c-11eb-9c82-c87884fe8d48.png">
3.
<img width="842" alt="Screen Shot 2021-05-19 at 9 59 34 AM" src="https://user-images.githubusercontent.com/5081226/118772329-4d7e7f00-b88c-11eb-967f-160d2d2eaf7a.png">